src/PrestaShopBundle/Controller/Admin/TranslationsController.php

```diff
@@ -222,7 +222,7 @@ protected function getTranslationsCatalogue(Request $request)
 
         $locale = $this->langToLocale($lang);
 
-        if (!is_null($theme)) {
+        if (!is_null($theme) && 'themes' === $type) {
```

Comment:
Isn't it the same as: https://github.com/PrestaShop/PrestaShop/pull/6353/files ?